src/mobile/src/main/java/tupperdate/android/appbars/mainTopBar.kt

```diff
@@ -1,27 +1,43 @@
 package tupperdate.android.appbars
 
-/*
-Commented because it doesn't find Modifier.weight, which however works in Onboarding.kt...
+import androidx.compose.foundation.Icon
+import androidx.compose.foundation.Text
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.IconButton
+import androidx.compose.material.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import tupperdate.android.R
 
 @Composable
-fun mainTopBar() {
+fun mainTopBar(
+    onChatClick: ()->Unit,
+    onProfileClick: ()->Unit,
+    modifier: Modifier=Modifier
```

Comment:
`modifier` is never passed to `TopAppBar` rendering it useless

Reply:
Yeah I removed